cozy.example.yaml

```diff
@@ -76,6 +76,10 @@ mail:
 # directory with the hooks scripts - flags: --hooks
 hooks: ./scripts/hooks
 
+# Auto updates scheduler
+auto_updates:
+  schedule: "@cron 0 0 0 * * *"
```

Comment:
For a cluster, only one node should have the `auto_updates.schedule`? Or should all node have it?

Reply:
For a cluster, I think it is better to control the auto updates via the administration API to have better supervision of when starting the autoupdates.